enacts/crop_suitability/maproom_crop_suit.py

```diff
@@ -554,16 +561,27 @@ def set_colorbar(
     data_choice,
 ):
     if data_choice == "suitability_layer":
+        colormap = pingrid.ColorScale(
+            "crop_suit",
+            [BROWN, BROWN, ORANGE, ORANGE, YELLOW, YELLOW,
+                PALEGREEN, PALEGREEN, GREEN, GREEN, DARKGREEN, DARKGREEN],
+            [0, 0.5, 0.5, 1.5, 1.5, 2.5, 2.5, 3.5, 3.5, 4.5, 4.5, 5],
+        )
```

Comment:
Instead of copy-pasting, define this once and assign to a variable that can be used in multiple places.